src/handlers/testing/exec/util/run-manager.ts

```diff
@@ -209,6 +221,27 @@ export class RunManager {
     return runId;
   }
 
+  private async createShareUrl(args: { testExternalId: string }) {
+    // Share URL is only applicable for local runs
+    // CI runs are always shareable
+    if (this.isCI || !this.includeShareUrl) {
+      return;
+    }
+    try {
+      const runId = this.currentRunId({ testExternalId: args.testExternalId });
+      const { shareUrl } = await this.post<{ shareUrl: string }>(
```

Comment:
I figured we'd just add a new flag to the /start endpoint and create everything there?

Reply:
Oh I see, we have no way to share an entire run?

Reply:
we'd have to update the webapp to be able to handle that and probably update the test metadata schema. it's setup right now so you can share a filtered view from the UI - https://github.com/autoblocksai/autoblocks/blob/main/apps/webapp/src/server/api/routers/testing.ts#L469